samples/meetings-transcription/nodejs/README.md

```diff
@@ -0,0 +1,130 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates how to get Transcript using Graph API and show it in the task module.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+- javascript
+extensions:
+contentType: samples
+createdDate: "10-06-2022 00:15:15"
+---
+
+# Bot to show the transcript of the meeting using Microsoft Graph API.
+
+This is a sample application which demonstrates how to get Transcript using Graph API and show it in the task module.
+
+## Key features
```

Comment:
Add 4 steps with screenshots including how to start transcript from the meeting.

Reply:
Added